src/js/tech/tech.js

```diff
@@ -339,13 +395,13 @@ class Tech extends Component {
    */
 
   textTracks() {
-    this.player_.textTracks_ = this.player_.textTracks_ || new TextTrackList();
-    return this.player_.textTracks_;
+    this.textTracks_ = this.textTracks_ || new TextTrackList();
```

Comment:
Instead what I did is when unloading a tech, I save the textTracks and give it back to the next tech with an option